app/models/application_draft.rb

```diff
@@ -9,6 +9,11 @@ class ApplicationDraft < ActiveRecord::Base
   STUDENT0_CHAR_LIMITED_FIELDS = Student::CHARACTER_LIMIT_FIELDS.map { |m| "student0_#{m}" }
   STUDENT1_CHAR_LIMITED_FIELDS = Student::CHARACTER_LIMIT_FIELDS.map { |m| "student1_#{m}" }
 
+  PROJECT1_FIELDS = %i(project1 plan_project1 why_selected_project1)
```

Comment:
Just a minor thing in terms of keeping the project accessible:  `%i` isn't widely used. Since it just saves a few extra colons, I'd vote for being more explicit and write the literal symbols instead.